.github/pull-request-template.md

```diff
@@ -1,40 +1,41 @@
 ## **Description**
-_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
-_1. What is the reason for the change?_
-_2. What is the improvement/solution?_
+
+<!--
+Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:
+1. What is the reason for the change?
+2. What is the improvement/solution?
+-->
+
+## **Related issues**
+
+Fixes: #
 
 ## **Manual testing steps**
 
-_1. Step1:_
-_2. Step2:_
-_3. ..._
+1. Go to this page...
+2.
+3.
 
 ## **Screenshots/Recordings**
 
-_If applicable, add screenshots and/or recordings to visualize the before and after of your change._
+<!-- If applicable, add screenshots and/or recordings to visualize the before and after of your change. -->
 
 ### **Before**
 
-_[screenshot]_
+<!-- [screenshots/recordings] -->
 
 ### **After**
 
-_[screenshot]_
-
-## **Related issues**
-
-_Fixes #???_
+<!-- [screenshots/recordings] -->
 
 ## **Pre-merge author checklist**
 
 - [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
-- [ ] I've clearly explained:
-  - [ ] What problem this PR is solving.
-  - [ ] How this problem was solved.
-  - [ ] How reviewers can test my changes.
-- [ ] I’ve indicated what issue this PR is linked to: Fixes #???
-- [ ] I’ve included tests if applicable.
-- [ ] I’ve documented any added code.
```

Comment:
I'm just hesitant on removing this "documentation" checkbox, as it seems like a good reminder.
Curious to hear what reviewers think of it.

Reply:
If we want to keep this, I feel like we ought to be more clear about what _kind_ of documentation contributors should add. There are different kinds of documentation, and some people aren't as eager to add documentation as others, so they might need some pointers. Perhaps we should at least recommend adding JSDocs?

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/21377/commits/a5ece18b64336229894b76faf20294266e309752